### ISSUE TITLE
Can't seem to get portaling and occlusion to work UE5

### ISSUE THREAD

**Issue by landervanregenmortel** [microsoft/ProjectAcoustics#117]
I've been experimenting with Project Acoustics today in a prototype of a project. My level consists of a dense network of hallways and rooms. I have followed the PA microsoft documentation for the setup. Somehow I only seem to get my reverb to work, and not my occlusion or portaling. I assume the portaling is computed correctly, since the debug arrows of the sound direction point towards the correct direction the sound should be coming from, however the actual audio spatialisation just travels through walls and doesn't keep occlusion in mind. Wether theres an object between me and the audio source or not, the result sounds the same.

I'm not sure what I'm missing or doing wrong in my setup, but I'm sure there's something. Any pointers are appreciated!

Also I'm new to github, I think I've posted this in the correct section but apologies in advance if I've made a mistake here.

Quick demonstration: https://www.youtube.com/watch?v=AfLDc3uLch0

**Comment by KyleStorck** [comment]
For spatialization, you need to have a spatialization plugin chosen, and have Spatialization Method set to Binaural. Unfortunarely this doesn't work with built-in Unreal spatialization. For spatialization, you can use the built-in Resonance spatializer. Double check these settings first.
![image](https://user-images.githubusercontent.com/1233941/191325619-b94e584f-2ecf-4377-a196-dbbde4bd681b.png)

![image](https://user-images.githubusercontent.com/1233941/191325547-54f3d534-5283-4e61-93f4-964d0f10b2fa.png)

As far as occlusion not being heard when you move behind the wall, that is weird. The acoustic parameters being returned say there is no occlusion. You can see the "Occlusion" value in yellow on your Debug Params stays zero when you move behind the wall. I'm not sure why. Your ceiling, walls, and floors are properly voxelized. . @MikeChemi Any thoughts on the occlusion?

**Comment by MikeChemi** [comment]
The dry values are varying, so I suspect occlusion is working and all the surfaces are just very reflective. Even the wet level has quite the boost which is what I would expect if you were in a fully enclosed reflective box.

**Comment by landervanregenmortel** [comment]
@KyleStorck Aha awesome! Correct, spatialisation works correctly now! Funny, because I had tried changing both the spatialisation plugin and method to the correct settings, but not both at the same time.


> The dry values are varying, so I suspect occlusion is working and all the surfaces are just very reflective. Even the wet level has quite the boost which is what I would expect if you were in a fully enclosed reflective box.

This seems to be correct as well. I rebaked after changing my surfaces from the marble preset to a very absorbing material and now the occlusion value is way more sensitive. I'm not entirely sure I understand why the dry value stays high in a reflective environment, when the player is far from the sound source several corridors away. Wouldn't the vast majority of the sound at that point be reverberated and reflected sound (which is what I assume the wet value represents), or am I misunderstanding how the dry/wet values work?

Thank you both for your help!
